lib/collection/src/update_handler.rs

```diff
@@ -61,6 +61,7 @@ pub struct OperationData {
 
 /// Signal, used to inform Updater process
 #[derive(Debug)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
`Operation` is the largest variant but also the most used by far, I don't think it is necessary to Box here

Reply:
Completely agree :heavy_check_mark: